GoogleDataTransport/GDTCORLibrary/GDTCOREvent.m

```diff
@@ -68,15 +58,26 @@ - (nullable instancetype)initWithMappingID:(NSString *)mappingID target:(NSInteg
   if (mappingID == nil || mappingID.length == 0 || target <= 0) {
     return nil;
   }
+  __block NSNumber *eventID;
+  dispatch_semaphore_t sema = dispatch_semaphore_create(0);
+  [GDTCOREvent nextEventIDForTarget:target
+                         onComplete:^(NSNumber *_Nullable newEventID) {
+                           eventID = newEventID;
```

Comment:
`newEventID` is nullable, but property `eventID` is marked as `nonnull`. Should we fail initialization in this case?

Reply:
Fixed.